v63003/u_inref/gtm8680.csh

```diff
@@ -17,6 +17,7 @@
 # Lockspace is 60000 to ensure we never call the garbage collector, which would clean up
 # the unused lock names from shared memory and speed up future lock commands in versions that
 # do not have the fix
+unsetenv gtm_db_counter_sem_incr
```

Comment:
Add comment "# to avoid counter semaphore overflow causing leftover shm as we need standalone access for later MUPIP SET LOCK_SPACE"